nvm.sh

```diff
@@ -2277,12 +2307,16 @@ nvm_download_artifact() {
     nvm_err "Removing the broken local cache..."
     command rm -rf "${TARBALL}"
   fi
+
   nvm_err "Downloading ${TARBALL_URL}..."
-  nvm_download -L -C - "${PROGRESS_BAR}" "${TARBALL_URL}" -o "${TARBALL}" || (
+  download_result=$(nvm_download_v2 -L -C - "${PROGRESS_BAR}" "${TARBALL_URL}" -o "${TARBALL}")
+  nvm_err "Download Http Status ${download_result}"
```

Comment:
i think we don't want to print this in the common success case, and probably not even in the error case. You can, however, wrap it in a NVM_DEBUG check for debugging:
```suggestion
  if [ "${NVM_DEBUG-}" = 1 ]; then
    nvm_err "Download Http Status ${download_result}"
  fi
```